File: examples/db/truncate/go/truncateTable.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Truncate the records in a table
func TruncateTable() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.Truncate(&db.TruncateRequest{
		Table: "users",
	})
	fmt.Println(rsp, err)
}

File: examples/db/truncate/node/truncateTable.js
const { DbService } = require("m3o/db");

// Truncate the records in a table
async function truncateTable() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.truncate({
    table: "users",
  });
  console.log(rsp);
}

truncateTable();
